CHANGELOG.md

```diff
@@ -4,20 +4,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
```

Comment:
I'd suggest removing this empty line since other titles and context are together

Reply:
Thanks. Updated.

Reply:
Resolving this conversation.